lib/launchers/dns_utils.rb

```diff
@@ -0,0 +1,71 @@
+#Define two functions to creat/delete dns records, used for osp and vsphere platform
+def create_dns_records(dns_solution, instances_name_prefix, ip_api, ip_api_int, ip_apps)
```

Comment:
This function is copied directly from v4 flexy-template, I do not think that would work in 3.x.  In 3.x, caller can determinate what A records will be created.
```
          dns_record = "#{task[:prefix]}.#{dns_component}"
          logger.info "Creating '#{dns_record}' record for: #{ips.join(?,)}"
          fqdn = dyn.dyn_replace_a_records(dns_record, ips)
```